<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormVenda</class>
 <widget class="QMainWindow" name="FormVenda">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>814</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Iniciar Venda</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../img/loja.png</normaloff>../../img/loja.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="produtoComboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>30</y>
      <width>241</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelProduto">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Produto:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="qtdeSpinBox">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableVenda">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>601</width>
      <height>192</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>CODPROD</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>DESCRICAO</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>QUANTIDADE</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VALOR UNIT.</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>VALOR TOTAL</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="btnIncluir">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Incluir</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/add.png</normaloff>../../img/add.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="labelTotal">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>290</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="editTotal">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>290</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelPagto">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>390</y>
      <width>147</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Forma de Pagamento:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRemover">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>160</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Remover</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/remove.png</normaloff>../../img/remove.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFinalizarVenda">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>470</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Finalizar Venda</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../img/greencheck.png</normaloff>../../img/greencheck.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="labelValorPago">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>420</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Valor Pago:</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="valorPagodoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>420</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="labelCliente">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>47</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Cliente:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="clienteComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>360</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QComboBox" name="formapgtoComboBox">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>Ã€ VISTA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>A PRAZO</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>produtoComboBox</tabstop>
  <tabstop>qtdeSpinBox</tabstop>
  <tabstop>btnIncluir</tabstop>
  <tabstop>tableVenda</tabstop>
  <tabstop>btnRemover</tabstop>
  <tabstop>editTotal</tabstop>
  <tabstop>clienteComboBox</tabstop>
  <tabstop>formapgtoComboBox</tabstop>
  <tabstop>valorPagodoubleSpinBox</tabstop>
  <tabstop>btnFinalizarVenda</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
